lib/tenstorrent/bh_arc/pcie.c

```diff
@@ -468,14 +468,14 @@ static int pcie_init(void)
 	if (IS_ENABLED(CONFIG_TT_SMC_RECOVERY)) {
 		pci0_property_table = (FwTable_PciPropertyTable){
 			.pcie_mode = FwTable_PciPropertyTable_PcieMode_EP,
-			.num_serdes = 2,
+			.num_serdes = 1,
```

Comment:
Regarding the change itself, I guess the issue is P300 specific, because there we need to use a more narrow PCIe lane width? For P300 the main issue would be that we setup both PCIe1 and PCIe0, right? In production firmware each ASIC only sets up one PCIe instance